packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -58,12 +58,12 @@ var dasherize = Ember.String.dasherize;
               "data": [
                 {
                   "id": "3",
-                  "type": "players"
+                  "type": "player"
```

Comment:
I think we should document this difference in some other places. Not everyone reads the guides.